### PR TITLE
41 tslint eslint

### LINKED ISSUE #41: Replace TSLint with ESLint

### PR DESCRIPTION
**Pull request by Macavirus**
closes #41

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,16 +23,21 @@
...
     "semantic-release": "^15.12.1",
     "ts-jest": "^23.10.5",
     "ts-node": "^7.0.1",
-    "tslint": "^5.11.0",
     "typescript": "^3.1.6"
   },
   "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^2.15.0",
+    "@typescript-eslint/parser": "^2.15.0",
+    "eslint": "^6.8.0",
```
Comment (vic08) [inline package.json]: should these be in dev dependencies?

**Comment by vic08** [comment]
why changing code style? i don't care much, but then the rest of app related packages should be updated as well. what's the motivation behind it?

**Comment by Macavirus** [comment]
I don't care too much about the coding style so long as there is one. It's really nice to have format on save work and decrease line noise in diffs. I think only major difference here was preferring single quote and no semis over double quotes and semis. Can revert if you like.

File: package.json (hunk 1 above)
Comment (Macavirus) [reply package.json]: Yes, should be

**Comment by vic08** [comment]
ok makes sense.

tslint was used in travis.yml and package.json scripts. So those should be updated as well

**Comment by Macavirus** [comment]
Planning to grab styles from apps as theirs seems the most consistently applied. Will replace this PR probably.

Also for esling + prettier see .e.g https://github.com/prettier/prettier-eslint/issues/101#issuecomment-313233479 for justification. Eslint will not fix long lines for one example (and then will fail lint in CI).  There are probably other ways. I could go either way on it, but eslint is less idempotent than eslint + prettier.

**Pull request closed without merging**